### PR TITLE
Side by side

### PR DESCRIPTION
**Pull request by brittgalloway**
## Side By Side Layout

I had issues with the PortableText component and adding styles via the style tag in the component (such as p and a tags), but no issues with styling from the global CSS sheet. So I added some specific text styles there.

I also wrapped the image in a figure because the figma seemed to have an image caption, so I added some placeholder text.

Lastly, I found a little bug with the nav when you drag the viewport smaller from desktop to mobile. It did a weird pop and scaled down. I tweaked the related transition property, and it's fixed now.

### CODE REVIEW

File: src/components/Hero.astro [hunk 1]
```diff
@@ -5,33 +5,42 @@ import cnpicon from '../images/cnp-icon.svg';
 const { heroText, heroImage, ctaText, ctaLink } = Astro.props;
 ---
 
+<div class="background">
+  <img src={heroImage} alt="decorative" class="full-img" />
+  <div class="content">
+    <PortableText value={heroText} />
+  </div>
+  {
```
Comment (daleinen7) [inline src/components/Hero.astro]: As per meeting discussion can you move this in the "content" block and style it to look like the CTA?

File: src/style/global.css [hunk 2]
```diff
@@ -59,3 +59,39 @@ h6 {
...
+  color: var(--black);
+  text-decoration-color: var(--black-transparent);
+  transition: text-decoration-color, color var(--transition);
+}
+a:hover {
+  color: var(--yellow);
+  text-decoration-color: var(--yellow);
+}
+
+/* Side by side PortableText sytles */
```
Comment (daleinen7) [inline src/style/global.css:124]: Would it make more sense to leave these in the side by side component?

File: src/style/reset.css [hunk 3]
```diff
@@ -123,3 +123,7 @@ table {
   border-collapse: collapse;
   border-spacing: 0;
 }
+p:empty,
```
Comment (daleinen7) [inline src/style/reset.css:126]: I need to steal this for all my projects. Makes so much sense.

File: src/components/SideBySide.astro [hunk 4]
```diff
@@ -4,32 +4,74 @@ import PortableText from './PortableText.astro';
...
+<div class="side-by-side wrapper">
+  <h3>{headline}</h3>
   <div class="info">
-    <h3>{headline}</h3>
-    <PortableText portableText={text} />
+    <PortableText portableText={text}/>
   </div>
+  <figure>
+    <img src={image} alt="Placeholder image" />
+    <figcaption>shoule we have a caption?</figcaption>
```
Comment (daleinen7) [inline src/components/SideBySide.astro]: Mispelled "should".

jk 

el oh el.

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Screenshot 2024-11-11 at 6 40 40 PM
Not a full on request, but what do you think about making the image full width of the content container on tablet and mobile?

Screenshot 2024-11-11 at 6 41 59 PM
On mobile can we narrow the margins? 

I guess that'd be done in the global wrapper section?

File: src/style/global.css (hunk 2 above)
Comment (daleinen7) [reply src/style/global.css:124]: nevermind. saw your comment.

File: src/style/reset.css (hunk 3 above)
Comment (brittgalloway) [reply src/style/reset.css:126]: it's great right???

File: src/components/Hero.astro (hunk 1 above)
Comment (brittgalloway) [reply src/components/Hero.astro]: done

**Review by daleinen7 — APPROVED** [review]

**Pull request merged**